src/lib.rs

```diff
@@ -1181,6 +1197,26 @@ impl<'a> WebViewBuilder<'a> {
     })
   }
 
+  /// Set whether background throttling should be disabled.
+  ///
+  /// By default, browsers throttle timers and even unload the whole tab (view) to free resources after roughly 5 minutes when
+  /// a view became minimized or hidden. This will permanently suspend all tasks until the documents visibility state
+  /// changes back from hidden to visible by bringing the view back to the foreground.
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Linux / Windows / Android**: Unsupported. Workarounds like a pending WebLock transaction might suffice.
+  /// - **iOS**: Supported since version 17.0+.
+  /// - **macOS**: Supported since version 14.0+.
+  ///
+  /// see https://github.com/tauri-apps/tauri/issues/5250#issuecomment-2569380578
+  pub fn with_background_throttling(self, policy: Option<BackgroundThrottlingPolicy>) -> Self {
+    self.and_then(|mut b| {
+      b.attrs.background_throttling = policy;
+      Ok(b)
+    })
+  }
```

Comment:
```suggestion
  pub fn with_background_throttling(self, policy: BackgroundThrottlingPolicy) -> Self {
    self.and_then(|mut b| {
      b.attrs.background_throttling = Some(policy);
      Ok(b)
    })
  }
```

I assume the user calls `with_background_throttling` when they decide to set a non-default value, so `Option<>` may not be necessary here.

Reply:
@pewsheen changed it accordingly, thank you!